src/main/java/in/org/projecteka/hiu/HiuConfiguration.java

```diff
@@ -299,8 +304,19 @@ public PatientConsentService patientConsentService(
                 consentRepository,
                 patientConsentRepository,
                 gatewayServiceClient,
-                healthInfoStatus);
+                healthInfoStatus,
+                patientConsentCertService);
+    }
 
+    @Bean
+    public PatientConsentCertService hiuConsentCertService(KeyPair keyPair) {
```

Comment:
Let's call it `PatientHIUCert` instead of `PatientConsentCert` because it is for whole server.

Reply:
done